index.html

```diff
@@ -1059,24 +1220,35 @@ <h2>Evidence</h2>
 }
         </pre>
 
+        <p class="note">
+For the avoidance of doubt, the <em>evidence</em> <a>property</a> provides
+information that is different from and complementary to the
+<a href="#proofs-aka-signatures">proof</a> property. The <em>evidence</em>
+property is used to express supporting information, such as documentary
+evidence, related to the integrity of the credential. In contrast, the
+<em>proof</em> property is used to express machine-verifiable mathematical
+proofs related to the integrity of the credential.
```

Comment:
Not just the integrity of the credential but it is also used to prove the authenticity of the credential's issuer.

Reply:
Fixed in dc3b3b19cc8ad888d81bedcb932130ffe38825d6.